mirrord/layer/src/file/hooks.rs

```diff
@@ -585,6 +601,17 @@ unsafe extern "C" fn fstatat_detour(
         .unwrap_or_bypass_with(|_| FN_FSTATAT(fd, raw_path, out_stat, flag))
 }
 
+#[hook_guard_fn]
+unsafe extern "C" fn fstatfs_detour(fd: c_int, out_stat: *mut statfs) -> c_int {
```

Comment:
Shouldn't we check that `*mut statfs` is a valid pointer:

> [EFAULT]		The sb argument	points to an invalid address.

Reply:
Maybe implement [`CheckedInto`](https://metalbear-co.github.io/mirrord/mirrord_layer/common/trait.CheckedInto.html) for `*mut statfs`?